README.md

```diff
@@ -66,7 +66,60 @@ By definition, members work on their respective squad, although they are free to
 
 ## 4. Release process
 
-// Work in progress ⚠️
+### Release engineer as a role
+
+  The release engineer responsibilities are, but not limited to:
+
+ 1. Own the entire release process (step-by-step).
+ 2. Provide visibility, at all stages, of the release to the wider audience (e.g. squad, tribe and iOS chapter lead).
+ 3. Collaborate with QA by:
+    1. Providing visibility to potential blockers to the wider audience.
+    2. Escalating abnormal influx of bugs, so the release, as a whole, can be reassessed.
+4. Making sure they can dedicate enough time for the release. In case this is not possible (due to other squad commitments), please inform the iOS chapter lead.
+
+### Release calendar
+
+Effort to release should be broken down by:
+
+1. Automated QA effort (e.g. `5h`)
+2. Manual QA effort (e.g. `3h`)
+3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`)
+4. Total effort
+
+
+| Version                  | Release Engineer(s)              | QA effort              | Engineering effort | Total effort |
+|--------------------------|----------------------------------| ---------------------- |--------------------|--------------|
+| 3.2.0                    | Danilo Aliberti                  | Automated: `12h53`<br>Manual: `10h`<br>| `UA-8166: 4h`<br>`UA-8149: 2d`<br>`UA-8187: 3h`<br>| Total: **3d6h** |
+
+### Release step-by-step
+
+1. Cut a release branch when the sprint ends (typically when the new sprint starts on Monday). Create a new branch from develop and push to origin (e.g `release/3.2.0`).
+2. Bump the release version by triggering its command (eg. `/release babylon:3.2.0`) in `#ios-builds` (you can run the command every time you want to upload a new build).
+    1. This creates a test Tesflight build (try to do one as early as possible, so that you can catch issues like missing/expired certificates or profiles and any other production build errors early)
+3. Trigger a hockey build from that branch.
+4. Testers will then begin their work against the build you just created.
+5. Any hotfix should target that branch, you, as the release engineer, are responsible for double checking the hotfix PR is pointing to the release branch (instead of develop). 
```

Comment:
```suggestion
5. Any hotfix should target that branch, and you, as the release engineer, are responsible for double checking that the hotfix's PR is pointing to the release branch (instead of `develop`). 
```